app/models/manageiq/providers/redhat/infra_manager/provision/configuration.rb

```diff
@@ -27,7 +35,12 @@ def configure_container
       configure_cpu(rhevm_vm)
       configure_host_affinity(rhevm_vm)
       configure_network_adapters
-      configure_cloud_init
+
+      if rhevm_vm.platform.downcase.include?("windows")
```

Comment:
could it be selected on the user side? there is a chance that eventually cloud-init will work on windows too (it actually may already, if you provide the right file for it - we just don't do that in RHV)

Reply:
I can fetch both sysprep and cloud-init templates. User could decide what to choose but if we do not support it on the engine side I think it could be confusing for the user.

Reply:
we do not have a template, but we do support the same generic payload mechanism to supply files in a cdrom image. So if miq provide the right template it may work. Not sure if it is supported for other providers yet, but in general there is a desire to support cloud-init on windows too

Reply:
I understand but to add cloud-init for windows is quite easy from miq perspective. I am not sure whether it is correct order of implementing it. We should first have it in the engine and later expose it in miq.

Does it mean that a customization template available in miq is good enough for the engine to consume?

Reply:
not sure we'll be investing into it in rhv any time soon(there is an rfe, but it's not targeted). 

i have no idea if that customization template works, it's simply different than the one we have in rhv. the mechanism is mostly passthrough, so it's not like it matters to rhv what you pass in. But whether it works in all windows versions it's hard to tell. We did have(and still do) quite a lot of issues across various versions
The only way how you can "delegate" this responsibility to rhv would be not to use custom-supplied templates but rely only on parameters exposed by rhv - downside is that it limits the features

Reply:
I think that we should avoid delegating this to RHV is something we should avoid, if possible. Instead we should pass directly to RHV whatever the ManageIQ, so it will be in the hands of the ManageIQ user to assemble the right Unatend.xml (or whatever) file for his particular version of Windows. Otherwise whenever we have problems in RHV they will be automatically be problems in ManageIQ as well.

Reply:
I would not like to limit it but let the user to pass anything it is needed. Maybe we should provide rhv specific template for the user which could be customized and passed to rhv.

Reply:
that is fine with me, it's just important to acknowledge the difference between what RHV supports and what this PR adds to MIQ - that it doesn't cover the exact same feature set.

Reply:
Do you mean to update PR description to state it clearly what are the differences?

Reply:
I think could-init or sysprep is fine for this PR.  In theory, I could see people wanting to do kickstart and cloud-init as well, but I don't think our dialogs are set up to allow selection of multiple templates yet.  This could be a follow-up set of PR's cc @gmcculloug